lib/manager/gradle-wrapper/artifacts.spec.ts

```diff
@@ -19,8 +19,6 @@ const config = {
   toVersion: '5.6.4',
 };
 
-jest.mock('../../platform');
```

Comment:
Is this related?

Reply:
no, simply not nessessary, because platform is always mocked by default in `test/globals.ts`

Reply:
removed, will be removed in #6233 anyways